modules/gapi/include/opencv2/gapi/ocl/core.hpp

```diff
@@ -5,23 +5,24 @@
 // Copyright (C) 2018 Intel Corporation
 
 
-#ifndef OPENCV_GAPI_GPU_CORE_API_HPP
-#define OPENCV_GAPI_GPU_CORE_API_HPP
+#ifndef OPENCV_GAPI_OCL_CORE_API_HPP
+#define OPENCV_GAPI_OCL_CORE_API_HPP
 
 #include <opencv2/core/cvdef.h>     // GAPI_EXPORTS
 #include <opencv2/gapi/gkernel.hpp> // GKernelPackage
 
 namespace cv {
 namespace gapi {
 namespace core {
-namespace gpu {
+#define gpu ocl
```

Comment:
This change is dangerous!

Provide empty "gpu" namespace for compatibility with inlined "ocl" namespace.

Reply:
Yes. Sure! I used it as a draft for fast prototyping, to understand the size of the problem.